subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/dependency_resolution.adoc

```diff
@@ -248,6 +261,37 @@ If multiple incompatible Gradle versions are in play, all should be used when se
 | `modules-2`           | `files-2.1`           | `metadata-2.90`           | Gradle 6.1
 |===
 
+[[sub:shared-readonly-cache]]
+=== Sharing the dependency cache with other Gradle instances
+
+Instead of <<sub:cache_copy,copying the dependency cache into each container>>, it's possible to mount a shared, read-only folder that will act as a dependency cache for all containers.
+This cache, unlike the classical dependency cache, is accessed without locking, making it possible for multiple builds to read from the cache concurrently.
+When Gradle will look for dependencies (artifacts or metadata), it will look into the classic writable cache, but will also search into the read-only cache.
+It means that if an artifact is present in the read-only cache, it will not be downloaded, but if it's missing from the read-only dependency cache, it will be added to the _local_, writable cache of the Gradle instance.
```

Comment:
```suggestion
If an artifact is present in the read-only cache, it will not be downloaded.
If an artifact missing from the read-only dependency cache, it will be added to the _local_, writable cache of the Gradle instance.
```